pyproject.toml

```diff
@@ -11,6 +11,7 @@ python = "^3.10"
 
 [tool.poetry.group.docs.dependencies]
 langgraph = { path = "libs/langgraph/", develop = true }
+langgraph-checkpoint = { path = "libs/checkpoint/", develop = true }
```

Comment:
what is this one used for?

Reply:
just for docs building (so that the references for checkpointers are valid)